blog/authors.yml

```diff
@@ -44,4 +44,28 @@ jayesh:
   name: Jayesh Potlabattini
   title: Web Team Lead | Typescript Enthusiast | Full Stack Web Developer
   url: https://github.com/JayeshVP24
-  image_url: https://github.com/JayeshVP24.png
\ No newline at end of file
+  image_url: https://github.com/JayeshVP24.png
+
+kunal:
+  name: Kunal Agrawal
+  title: President - Mind Benders | SSoC'22
+  url: https://github.com/kunalagra
+  image_url: https://github.com/kunalagra.png
+
+keval:
+  name: Keval Waghate
+  title: Publication Head - Mind Benders
+  url: https://github.com/KevalW
+  image_url: https://github.com/KevalW.png
+
+deexith:
+  name: Deexith Madas
+  title: Treasurer -  Mind Benders
+  url: https://github.com/MDeexith
+  image_url: https://github.com/MDeexith.png
+
+ananta:
+  name: Ananta Pandey
+  title: Jt. Publication Head - Mind Benders
+  url: https://github.com/Ananta564
+  image_url: https://github.com/Ananta564.png
```

Comment:
I would highly recommend to add a pfp here, we discourage blanks

Reply:
Sure. Would Pass on the message to the team!